ol-fyi-local-infra/compose.yml

```diff
@@ -40,6 +40,17 @@ services:
       - 6379:6379
       # - 8001:8001
 
+  minio:
+    image: minio/minio
```

Comment:
specify the version [RELEASE.2024-05-10T01-41-38Z](https://hub.docker.com/layers/minio/minio/RELEASE.2024-05-10T01-41-38Z/images/sha256-ea0203747a6b779d26ceee879ff9c1d8b70c0d10196a4f969f8ceb4d1e3904bb?context=explore) for instance.

Reply:
Acknowledged and pushed